### ISSUE TITLE
Failure to obtain expected results in human DLPFC dataset

### ISSUE THREAD

**Issue by Bink98** [uhlerlab/STACI#2]
Hi, 
Thank you for this great work! I'm trying to analysis human DLPFC dataset (https://www.nature.com/articles/s41593-020-00787-0) using STACI , but I can't match the spatial clusters with the manual annotation in their manuscript following the provided tutorials. 
The data and code are at https://drive.google.com/drive/folders/1pOaZNSG_qcZvCNSJGpsWFuw2a4r3ktcF?usp=sharing
The result I obtained look like this:
<img width="1244" alt="image" src="https://github.com/uhlerlab/STACI/assets/24759898/a31995a0-baad-4395-9975-1690ed5e5b92">
 Are there any parameters  that need to be adjusted for this dataset?

**Comment by xinyiz98** [comment]
Thank you for your interests in our work. It looks like your current code did not incorporate the reconstruction loss of the adjacency matrix in training. You need to set useA=True to let the autoencoder loss include the reconstruction loss of the adjacency matrix, which is the default setup for our full model.  I have updated our code and readme to clarify this. 

Additionally, with useA=False as it is, I would expect the validation loss of gene expression to decrease with training. I noticed that you did not filter out any lowly expressed genes. Usually, we would do some filtering based on the expression level. For example, in our demo on the 10X visium AD brain data, we filtered down to 17186 input genes. A lower learning rate might also help.

It could also be good to make sure that the correct adjacency matrix is used by using the plotA function. 

Please feel free to let me know if the changes do not work and I'm happy to look further into this.

**Comment by Bink98** [comment]
It works. Thanks! By the way, what do the XreconWeight and ridgeL parameters mean? Do I need to change these parameters for different datasets?

**Comment by xinyiz98** [comment]
XreconWeight is the weight of the gene expression reconstruction loss. This is needed when adding together the losses of gene expression reconstruction, graph adjacency reconstruction, and KL divergence. I would set it large enough so that the reconstructed gene expression makes sense, e.g. cell type clusters can be recovered in the reconstructed gene expression. It should not be too large such that the contribution of the adjacency reconstruction loss becomes negligible.

ridgeL is the regularization weight of the gene dropout parameter Pi. This is defined in: https://www.nature.com/articles/s41467-018-07931-2 ("Furthermore, our model contains a tunable zero-inflation regularization parameter that acts as a prior on the weight of the dropout process. This is achieved using the ridge prior on the dropout probabilities and zero inflation (∏ parameter)").

**Comment by Bink98** [comment]
Thank you so much!
